src/__tests__/Slides.js

```diff
@@ -0,0 +1,81 @@
+import cases from 'jest-in-case';
+
+import { getHTML, shouldTransform } from '../Slides';
+
+cases(
+  'url validation',
+  ({ url, valid }) => {
+    expect(shouldTransform(url)).toBe(valid);
+  },
+  {
+    'non-Slides url': {
+      url: 'https://not-a-slides-url.com',
+      valid: false,
+    },
+    "non-Slides url ending with 'slides.com'": {
+      url: 'https://this-is-not-slides.com',
+      valid: false,
+    },
+    "non-Slides url ending with 'slides.com' and having '/embed/'": {
+      url: 'https://this-is-not-codepen.io/news/math/embed',
```

Comment:
`codepen.io`? 🤔

Reply:
its still an non-slides url, but i will fix that. :ok_hand:

Reply:
The purpose of this specific test is to ensure we're not doing `endsWith('slides.com')`